prepline_general/api/general.py

```diff
@@ -147,31 +147,47 @@ def pipeline_api(
     filename="",
     m_strategy=[],
     m_coordinates=[],
+    m_ocr_languages=[],
     file_content_type=None,
     response_type="application/json",
 ):
     strategy = (m_strategy[0] if len(m_strategy) else "fast").lower()
-    strategies = ["fast", "hi_res", "auto"]
+    strategies = ["fast", "hi_res", "auto", "ocr_only"]
     if strategy not in strategies:
         raise HTTPException(
             status_code=400, detail=f"Invalid strategy: {strategy}. Must be one of {strategies}"
         )
 
+    # Note (austin) - there is no fast mode for images
+    # In this case we need to use hi_res
+    if file_content_type in ["image/jpeg", "image/png"]:
+        strategy = "hi_res"
```

Comment:
since the awkwardness of passing an irrelevant strategy for an image file type exists in the library itself in [partition()](https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/auto.py), i think this kind of logic should live there, not in the API.

another reason is that unstructured-ingest would have the same issue using a single `strategy` across many doc types.